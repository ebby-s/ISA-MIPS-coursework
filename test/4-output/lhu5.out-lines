TB : INFO : register_v0=00000d5d
TB : finished; active=0
