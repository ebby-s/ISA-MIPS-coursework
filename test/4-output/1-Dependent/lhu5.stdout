Loading test/2-binary/1-Dependent/lhu5.hex.txt into RAM
VCD info: dumpfile mips_cpu_bus_tb.vcd opened for output.
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x261003e8
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x96020004
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x0000f808
TB : INFO : RAM_ACCESS: Read from 0x000003ec, data: 0x00000d5d
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x00000000
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x00000000
TB : INFO : register_v0=00000d5d
TB : Finished : active=0
